### PR TITLE
feat: adds home page

### PR DESCRIPTION
**Pull request by Ensar025**

### CODE REVIEW

**Comment by vanhoofmaarten** [comment]
Nice to see you provided an image for both dark and light mode. Are you already aware you can use CSS within `<style>` tags within SVGs? Can you try to replace all the JavaScript dark-mode detection and combine both SVGs to one that can detect dark-mode by itself?

**Comment by Ensar025** [comment]
> Nice to see you provided an image for both dark and light mode. Are you already aware you can use CSS within `<style>` tags within SVGs? Can you try to replace all the JavaScript dark-mode detection and combine both SVGs to one that can detect dark-mode by itself?

Oh, didn't know, sounds cool. I'll try to do that then

File: packages/frontend/src/views/HomeView.vue [hunk 1]
```diff
@@ -1,7 +1,40 @@
...
+        A lot of Open-Source projects suffer from financial sustainability and face the risk of not getting maintained
+        due to a lack of resources. Use the Dependency Health Monitor to view the financial state of your dependencies
+        and support your dependency!
+      </h4>
+      <Button @click="redirectToExecuteReportPage" label="Execute the report!" icon="pi pi-heart" />
+    </div>
+    <div id="help-improve" class="col-6">
+      <IconHelpImproveVue />
+      <div class="flex justify-content-center">
+        <Button
```
Comment (vanhoofmaarten) [inline packages/frontend/src/views/HomeView.vue:15]: Can you replace this with an `<a href="https://storyset.com/people" target="_blank" rel="noopener noreferer">`?

**Review by vanhoofmaarten — CHANGES_REQUESTED** [review]

File: packages/frontend/src/views/HomeView.vue (hunk 1 above)
Comment (Ensar025) [reply packages/frontend/src/views/HomeView.vue]: Resolved in a10417f5bc57f84d48c6ff15e83be98ba7ad8e6f

File: packages/frontend/src/views/HomeView.vue [hunk 2]
```diff
@@ -7,34 +7,24 @@
...
     <div id="help-improve" class="col-6">
       <IconHelpImproveVue />
       <div class="flex justify-content-center">
-        <Button
-          @click="openStorySetPage"
-          label="People illustrations by Storyset"
-          class="p-button-text"
-          icon="pi pi-external-link"
-        />
+        <a class="no-underline" href="https://storyset.com/people" target="_blank" rel="noopener noreferer">
```
Comment (vanhoofmaarten) [inline packages/frontend/src/views/HomeView.vue:17]: `Button` component will render a `button` HTML element, right? If so, this isn't allowed. The best thing we can do here, semantically, is copy the HTML output from the primevue `Button` component and replace the HTML `button` element with an `a` element

Comment (vanhoofmaarten) [reply packages/frontend/src/views/HomeView.vue:17]: See **Permitted Content** in https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#properties

**Review by vanhoofmaarten — CHANGES_REQUESTED** [review]

File: packages/frontend/src/views/HomeView.vue (hunk 2 above)
Comment (Ensar025) [reply packages/frontend/src/views/HomeView.vue]: Resolved in 7c4733217c8f0f384c72d18f886503816ff1a733

**Pull request merged**